packages/mux-player/src/media-theme-mux/styles.css

```diff
@@ -61,6 +61,12 @@
   width: 100%;
 }
 
+[disabled],
+[aria-disabled='true'] {
+  opacity: 60%;
+  cursor: not-allowed;
```

Comment:
(blocking): is this really needed? I feel it a bit much. like the fact that the cursor would not become a hand would be sufficient IMO